testdata/p4_16_errors_outputs/string-e1.p4-stderr

```diff
@@ -1,9 +1,15 @@
 string-e1.p4(2): [--Werror=type-error] error: Field 's' of 'struct S' cannot have type 'string'
     string s;
            ^
+string-e1.p4(2)
```

Comment:
These error messages make no sense to me.

Reply:
'string' is not an allowed type for a struct field. How would you write the error message?

Reply:
Why is the 
```
string-e1.p4(2)
    string s;
```
duplicated?

Reply:
First the message points to the field name, then to the type name.

Reply:
I guess it would be nice to add these labels, if possible. E.g., 
Field:  string s
Type: string s